x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/services/ingest.ts

```diff
@@ -19,12 +19,11 @@ import {
 import { GetPolicyListResponse, GetPolicyResponse, UpdatePolicyResponse } from '../../../types';
 import { NewPolicyData } from '../../../../../../../common/endpoint/types';
 
-const INGEST_API_ROOT = `/api/ingest_manager`;
+const INGEST_API_ROOT = `/api/fleet`;
```

Comment:
@paul-tavares @nchaulet thoughts on removing the hardcoded reference and using `epmRouteService` etc like this: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/hooks/endpoint/upgrade.ts#L10

That'd avoid having to find/replace these references in the future

Reply:
sound good to me if we can avoid hardcoded path :)

Reply:
yes, I agree that we (security solution) should stop using hardcoded values and start using the `const` exposed in ingest. @nchaulet we can take that one - you don't have to include it in your PR.